app/src/main/res/layout/activity_splash_screen.xml

```diff
@@ -8,20 +8,52 @@
     tools:context=".ui.splash.SplashScreenActivity"
 >
 
-    <TextView
-        android:id="@+id/textView"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="8dp"
-        android:layout_marginEnd="8dp"
-        android:layout_marginLeft="8dp"
-        android:layout_marginRight="8dp"
-        android:layout_marginStart="8dp"
-        android:layout_marginTop="8dp"
-        android:text="Splash Screen"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-    />
+  <TextView
+      android:id="@+id/tv_version_number"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="@dimen/dimen_60dp"
+      android:layout_marginEnd="@dimen/dimen_8dp"
```

Comment:
using names such as `dimen_8dp` defeats the purpose. Replace with something meaningful such as `splash_screen_tv_margin`

Reply:
@sakchhams Updated the resource names. Is it fine now?